src/Modal/styles.css

```
@@ -0,0 +1,47 @@
+.modal {
+  position: fixed;
+  top: 50%;
+  left: -50%;
```

```suggestion
  top: -50%;
  left: 50%;
```

top should be -50% instead of left :p